index.js

```diff
@@ -28,16 +28,21 @@ function convert(source) {
 
   for (let i = 0; i < lines.length; i++) {
     let line = lines[i];
-    if (line.charAt(0) === " ") {
+    //some lines are a continuation of the previous one & start with a ' ;'
+    if (line.substr(0, 2) === ' ;') {
+      const nextLine = lines.splice(i+1, 1);
+      line += nextLine;
+    }
```

Comment:
This behavior is similar to line starting with ` `? If so, the code should be similar to the line below.